### ISSUE TITLE
strapi-provider-upload-cloudinary - "Error uploading to cloudinary: Server returned unexpected status code - 403"

### ISSUE THREAD

**Issue by Rhym** [strapi/strapi#10825]
## Cloudinary upload returning 403

Using the upload provider [strapi-provider-upload-cloudinary](https://www.npmjs.com/package/strapi-provider-upload-cloudinary) in a fresh install of Strapi and the following config file:

```javascript
// ./config/plugins.js
module.exports = ({ env }) => {
  return {
    upload: {
      provider: "cloudinary",
      providerOptions: {
        cloud_name: env("CLOUDINARY_NAME"),
        api_key: env("CLOUDINARY_KEY"),
        api_secret: env("CLOUDINARY_SECRET"),
      },
      actionOptions: {
        upload: {
          folder: env("CLOUDINARY_FOLDER_NAME"),
        },
        delete: {},
      },
    },
  };
};
```

Returns an `Error uploading to cloudinary: Server returned unexpected status code - 403` response when I upload a file in the media library admin page (http://localhost:1337/admin/plugins/upload).

### Expected behaviour

The image will upload to Cloudinary.

### Code snippets

If applicable, add code samples to help explain your problem.

### System

- Node.js version: v14.15.4
- NPM version: 6.14.10
- Strapi version: 3.6.8
- Database: PostgreSQL
- Operating system: OSX

### Additional context

I have tried a different key/secret with no luck.

**Comment by cwray-tech** [comment]
Hi, have you set all the required keys in your .env file?

**Comment by Rhym** [comment]
Hey there, @cwray-tech Yes I have. Console logged them out in the `node_modules/strapi-provider-upload-cloudinary/lib/index.js` just to be sure too.

**Comment by derrickmehaffy** [comment]
403 means that whatever credentials you are setting either aren't being read or are not correct.

But if you are already logging them and they are showing as defined then the creds are wrong. I'd suggest contacting Cloudinary.

I just spun up a fresh instance and could not reproduce this.

**Comment by cwray-tech** [comment]
@Rhym yeah I would double check your cloudinary credentials. Also might want to see if you need to white list your domain in cloudinary.

**Comment by Rhym** [comment]
Hey guys, thanks for taking the time to look at this. I'll get in touch with Cloudinary and report back when I get everything fixed. Have a great week!

**Comment by derrickmehaffy** [comment]
Given how old this issue is, I'll set it as closed for now but if someone can reproduce on Strapi v4 please feel free to open a new issue and link back to this one. Thanks!
